File: src/co/jp/mypage.java
const button = document.getElementById("changeColorButton");
function changeColors() {
    const body = document.body;
    if (body.style.background === "blue") {
        body.style.background = "#f5f5f5"; 
        body.style.color
          = "blue";
    } else {
        body.style.background = "blue"; 
        body.style.color = "white";
    }
}

File: src/co/jp/mypage/mypage.js
let isOrigin = true;
function changeColor() {
  document.body.style.backgroundColor = isOrigin ? "lightblue" : "#fffacd";
  document.body.style.color = isOrigin ? "white" : "black";
  document.querySelector("a").style.color = isOrigin ? "white" : "black";
  document.querySelector("button").style.color = isOrigin ? "white" : "black";
  isOrigin = !isOrigin;
}